bin/create-docs.sh

```diff
@@ -42,9 +42,4 @@ echo "Creating man pages"
 Rscript -e ' libDir <- "../../lib"; library(R4ML, lib.loc=libDir); library(knitr); knit_rd("R4ML", links = tools::findHTMLlinks(paste(libDir, "R4ML", sep="/"))) '
 popd
 
-pushd R4ML/vignettes
-echo "Creating Vignettes"
-Rscript -e ' libDir <- "../../lib"; library(R4ML, lib.loc=libDir); library(devtools); devtools::build_vignettes()'
-popd
```

Comment:
@nilmeier we would like to have this so that in future when we add the vignette we don't have to modify it.

Can we just create the R4ML.md and puts in the logo and have this "
R4ML, is a open-source R package from IBM. R4ML provides a bridge between R and Apache SystemML, allowing R scripts to invoke custom algorithms developed in SystemML's R-like domain specific language. This capability also provides a bridge to the algorithm scripts that ship with Apache SystemML, effectively adding a new library of prebuilt scalable algorithms for R on Apache Spark. R4ML integrates seamlessly SparkR, so data scientists can use the best features of SparkR and SystemML together in the same script. In addition, the R4ML package provides a number of useful new R functions that simplify common data cleaning and statistical analysis tasks.
"